crates/bevy_macro_utils/src/fq_std.rs

```diff
@@ -33,14 +33,14 @@
 use proc_macro2::TokenStream;
 use quote::{quote, ToTokens};
 
-pub(crate) struct FQAny;
-pub(crate) struct FQBox;
-pub(crate) struct FQClone;
-pub(crate) struct FQDefault;
-pub(crate) struct FQOption;
-pub(crate) struct FQResult;
-pub(crate) struct FQSend;
-pub(crate) struct FQSync;
+pub struct FQAny;
```

Comment:
Doc strings would be nice for these, but I won't block on them.

Reply:
No that is a good point, I'll make some short documentation for each of the new public facing types.